plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -55,7 +56,10 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
     super('@opentelemetry/instrumentation-fs', VERSION, config);
   }
 
-  init(): InstrumentationNodeModuleDefinition<FS>[] {
+  init(): (
+    | InstrumentationNodeModuleDefinition<FS>
+    | InstrumentationNodeModuleDefinition<FSPromises>
+  )[] {
```

Comment:
nit: i think this type is more explicit

```suggestion
  init(): [InstrumentationNodeModuleDefinition<FS>, InstrumentationNodeModuleDefinition<FSPromises>] {
```